fenix/app/src/main/res/values/strings.xml

```diff
@@ -1829,12 +1829,8 @@
     <string name="addresses_edit_address">Edit address</string>
     <!-- Title of the "Manage addresses" screen -->
     <string name="addresses_manage_addresses">Manage addresses</string>
-    <!-- The header for the first name of an address -->
-    <string name="addresses_first_name">First Name</string>
-    <!-- The header for the middle name of an address -->
-    <string name="addresses_middle_name">Middle Name</string>
-    <!-- The header for the last name of an address -->
-    <string name="addresses_last_name">Last Name</string>
```

Comment:
We have a [specific process](https://github.com/mozilla-mobile/fenix/blob/main/docs/Working-with-Strings.md) for removing strings that we'll need to follow instead of just deleting these.

Reply:
TIL. Thanks. This is fixed in [7dd261c](https://github.com/mozilla-mobile/firefox-android/pull/5586/commits/7dd261cf41a513b1a49419016ba0a0b90d67629f)